Libs/MRML/Core/vtkMRMLStorageNode.h

```diff
@@ -26,6 +27,77 @@ class vtkStringArray;
 
 // STD includes
 #include <vector>
+#include <algorithm>
+
+///
+/// A class for recording warnings and errors associated with this
+/// vtkMRMLStorableNode.
+class VTK_MRML_EXPORT vtkMessageCollection : public vtkObject
```

Comment:
Since this is added to the public API, I suggest to move this into its own file.

Likely a file to be added to https://github.com/Slicer/vtkAddon

Reply:
Since `vtkAddon` is not part of `github.com/Slicer/Slicer`, might I leave `vtkMessageCollection` where it is until I can get an implementation into and released from `vtkAddon`?  Perhaps I should rename the current one as `vtkMRMLMessageCollection` so that when `vtkMessageCollection` is released from vtkAddon there won't be a naming conflict ... and then I can easily change the calls from `vtkMRMLStorageNode`, etc. to use the `vtkAddon` implementation and `vtkMessageCollection` naming.

Reply:
I will do the renaming to `vtkMRMLMessageCollection`.